pkg/monitors/http/http.go

```diff
@@ -146,6 +158,29 @@ func (m *Monitor) Shutdown() {
 	}
 }
 
+func (m *Monitor) forgeURL(site string) (normalizedURL *url.URL, err error) {
```

Comment:
Call this `normalizeURL` -- "forge" is what you do to hot iron.

Reply:
^^ sorry, my english ...